src/cbl-mariner/manifest.json

```diff
@@ -394,6 +394,32 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/ubuntu/18.04/cross/freebsd/13",
```

Comment:
The path should be updated to point to the cbl-mariner directory.

Reply:
Commenting on old commit :)